scripts/view_bench.py

```diff
@@ -0,0 +1,104 @@
+# %%
+import pandas as pd
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import random
+
+def random_color_generator():
+    r = random.randint(0, 255)
+    g = random.randint(0, 255)
+    b = random.randint(0, 255)
+    return (r / 255, g / 255, b / 255)
+
+def view_results(paths, units):
+    dfs = []
+    benchmark_names = []
+    for i, path in enumerate(paths):
+        # NOTE: make sure format() has been run
+        df = pd.read_csv(path, sep=",", header=None)
+        # remove first row 
+        df = df.iloc[1:]
+        if i == 0: 
+            benchmark_names = df[0]
+        else: 
+            if not (benchmark_names == df[0]).all():
+                print("Mismatched benchmark names")
+                return
+        dfs.append(df)
+    
+    colors = [random_color_generator() for _ in range(len(paths))]
+    for i in range(1, len(benchmark_names)+1):
+        plt.clf()
+        plt.title(benchmark_names[i] + f"{units}", wrap=True)
+
+        for df_i, df in enumerate(dfs):
+            benchmark_runtimes = df.T[1:][i]
+            # convert to milliseconds 
+            if units == 'ms':
+                benchmark_runtimes = benchmark_runtimes / 1_000_000
+            if units == 's':
+                benchmark_runtimes = benchmark_runtimes / 1_000_000_000
```

Comment:
this seems to be the source of the error in my previous comment
```
    benchmark_runtimes = benchmark_runtimes / 1_000_000
                         ~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~
```

Reply:
[e713769](https://github.com/Syndica/sig/pull/316/commits/e713769884cd166d46927ce9ddd31f00f7e621ac)